frontend/pages/club/[club]/edit.js

```diff
@@ -262,6 +251,24 @@ class ClubForm extends Component {
     })
   }
 
+
+  deleteEvent(id) {
+    console.log(this.state.club.code)
+    console.log(`/clubs/${this.state.club.code}/events/${id}`)
+    doApiRequest(`/clubs/${this.state.club.code}/events/${id}`, {
```

Comment:
Would prefer destructuring out the `code` before using it in the string interpolation